.github/workflows/main.yml

```diff
@@ -63,3 +81,25 @@ jobs:
           git -c user.name='ci' -c user.email='ci' commit -m init
           git push -f -q https://git:${{ secrets.github_token }}@github.com/${{ github.repository }} HEAD:gh-pages
         if: github.event_name == 'push' && github.event.ref == 'refs/heads/master'
+
+  msrv:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-14, windows-latest]
+    env:
+      MSRV: 1.63
```

Comment:
If you use `cargo-hack`, you don't need to track the MSRV in yet-another-place

See
- https://github.com/rust-lang/cargo/blob/b4bb1eacdb73f9ca3eb9d878e848e69b64db656d/.github/workflows/main.yml#L269-L270
- https://github.com/rust-lang/cargo/pull/13523

Reply:
Thanks, I've updated the PR.